bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -287,35 +283,34 @@ if do_injections:
     sites = [ifo[0] for ifo in ifos]
 
     # Triggers and injections recovered in some form: discard ones at vetoed times
-    found_trigs = ppu.load_injections(found_file, vetoes)
-    found_injs = ppu.load_injections(found_file, vetoes, sim_table=True)
+    inj_file = ppu.load_triggers(found_missed_file, vetoes)
 
     logging.info("Missed/found injections/triggers loaded.")
 
     # Extract columns of found injections
     found_inj = {}
-    found_inj['ra'] = np.asarray(found_injs.get_column('longitude'))
-    found_inj['dec'] = np.asarray(found_injs.get_column('latitude'))
-    found_inj['time'] = np.asarray(found_injs.get_column('geocent_end_time')) +\
-                        np.asarray(found_injs.get_column('geocent_end_time_ns')*\
-                                   10**-9)
-    found_inj['dist'] = np.asarray(found_injs.get_column('distance'))
+    found_inj['ra'] = inj_file['found/ra'][:]
+    found_inj['dec'] = inj_file['found/dec'][:]
+    found_inj['time'] = inj_file['found/tc'][:]
+    found_inj['dist'] = inj_file['found/distance'][:]
 
     # Extract columns of triggers
     found_trig = {}
-    found_trig['mchirp'] = np.asarray(found_trigs.get_column('mchirp'))
-    found_trig['bestnr'] = ppu.get_bestnrs(found_trigs, q=chisq_index,
-                                           n=chisq_nhigh,
-                                           null_thresh=null_thresh,
-                                           snr_threshold=snr_thresh,
-                                           sngl_snr_threshold=sngl_snr_thresh,
-                                           chisq_threshold=new_snr_thresh,
-                                           null_grad_thresh=null_grad_thresh,
-                                           null_grad_val=null_grad_val)
+    found_trig['mchirp'] = mchirp_from_mass1_mass2(
```

Comment:
Here you would then have two solutions.
1. opening the file as `r+`, creating the new dataset `'found/mchirp'`, and using it (and perhaps deleting it to restore the file as it was), or
2. storing this as an ad-hoc `found_injs_mchirp` to avoid modifying the files

Reply:
I also considered adding this to the file within `pycbc_multi_inspiral` but decided to only save the masses there and calculate mchirp here. Other scripts might be using these files at the same time, so writing to them here might be a problem. And if other scripts need this value, it may be better to move the calculation earlier in the workflow.

Reply:
Now that I'm going about this, I think doing this calculation earlier would be best (which means less work for this PR and more work for another). I'm willing to hear other opinions of course.